test/functional/apps/visualize/_tsvb_chart.ts

```diff
@@ -142,6 +148,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       after(async () => {
         await security.testUser.restoreDefaults();
         await esArchiver.unload('index_pattern_without_timefield');
```

Comment:
We should be able to remove this `await esArchiver.unload('index_pattern_without_timefield');` since the load empty_kibana deletes the .kibana index